levels/level_manager.go

```diff
@@ -1699,12 +1719,89 @@ func (lm *LevelManager) dump() {
 }
 
 func containsTable(seg *segment, tabID sst.SSTableID) bool {
-	for _, te := range seg.tableEntries {
-		if bytes.Equal(te.SSTableID, tabID) {
-			return true
+	n := len(seg.tableEntries)
```

Comment:
I don't think this is right. SSTableIDs are just random UUIDs and won't be in any special order in the segment. I think there's no alternative than iterating over all elements

Reply:
That's right, I'll go ahead and revert it back to the original